gate-oauth2/src/main/groovy/com/netflix/spinnaker/gate/security/oauth2/provider/GithubProviderTokenServices.groovy

```diff
@@ -39,7 +41,7 @@ class GithubProviderTokenServices implements SpinnakerProviderTokenServices {
   GithubRequirements requirements
 
   private String tokenType = DefaultOAuth2AccessToken.BEARER_TYPE
-  private OAuth2RestOperations restTemplate
+  OAuth2RestOperations restTemplate
```

Comment:
Why the increase in visibility?

Reply:
As explained above, in order to make the code testable the private modifier was removed from the instance variable `restTemplate` to allow dependency mocking. A better design would inject this dependency, but that would require changes of a much larger scope.

Reply:
Please add `@VisibleForTesting` then.

Reply:
@dbyron-sf The codebase is using two such annotations:

* `com.netflix.spinnaker.kork.annotations.VisibleForTesting`
* `com.google.common.annotations.VisibleForTesting`

Can you confirm which one should be used?

Reply:
Yeah, I wish we were consistent here.  I suppose the kork one to limit dependencies.  Not sure there's a strong argument either way.

Reply:
Change made as requested.